bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/WicketApplication.java

```diff
@@ -93,7 +111,7 @@ public void init() {
 
         optimizeForWebPerformance();
 
-        new AnnotatedMountScanner().scanPackage("de.agilecoders.wicket.samples.pages").mount(this);
```

Comment:
Actually this class wasn't dropped: https://github.com/wicketstuff/core/blob/master/annotation/src/main/java/org/wicketstuff/annotation/scan/AnnotatedMountScanner.java

my local build is green ... :(

I can run `bootstrap-samples` using `mvn jetty:run`

```
Apache Maven 3.9.1 (2e178502fcdbffc201671fb2537d0cb4b4cc58f8)
Maven home: /opt/maven
Java version: 17.0.8.1, vendor: Private Build, runtime: /usr/lib/jvm/java-17-openjdk-amd64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.15.0-86-generic", arch: "amd64", family: "unix"
```